docs/painless/painless-contexts/painless-runtime-fields-context.asciidoc

```diff
@@ -25,6 +36,10 @@ about how to use runtime fields.
 `void`::
         No expected return value.
 
+`extract`::
```

Comment:
This is a method on the type returned by `dissect` and `grok`. I think this part of the page is for the return value of the script.